unit_tests/log_transfer/test_main_chkpt_info.cpp

```diff
@@ -0,0 +1,800 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#define CATCH_CONFIG_MAIN
+#include "catch2/catch.hpp"
+
+#include "client_credentials.hpp"
+#define private public
+#include "log_checkpoint_info.hpp"
+#undef private
```

Comment:
In case of unit tests, imo it is better to separate the header of the functionality that is being tested.
```suggestion
// hack to access the private parts of log_checkpoint_info, to verify that the members remain the same after pack/unpack
#define private public
#include "log_checkpoint_info.hpp"
#undef private

#include "client_credentials.hpp"
```